x-pack/solutions/observability/plugins/apm/public/components/shared/trace_waterfall/toggle_accordion_button.tsx

```diff
@@ -23,14 +24,25 @@ export function ToggleAccordionButton({ isOpen, childrenCount, onClick }: Props)
       justifyContent="center"
       responsive={false}
       css={{ position: 'relative', display: 'flex', width: `${TOGGLE_BUTTON_WIDTH}px` }}
```

Comment:
Not part of this PR, but isn't `EuiFlexGroup` already setting the `display: 'flex'` 🤔